packages/frontend/src/components/profile/two_factor/TwoFactorAuth.js

```diff
@@ -136,7 +135,7 @@ const TwoFactorAuth = ({ twoFactor, history, isBrickedAccount, onDisableBrickedA
                         <FormButton
                             onClick={() => history.push('/enable-two-factor')}
                             trackingId="2FA Click enable button"
-                            disabled={!account.canEnableTwoFactor || existingContract}
+                            disabled={true}
```

Comment:
I think we should only disable it if it is NEAR WALLET not MNY. 
`isWhitelabel ? (!account.canEnableTwoFactor || existingContract) : false`
Leave existing logic as it is for MNY and only apply disable on NEAR WALLET?

Also instead of just disable button, would it be nicer to add simple tooltip, showing some context around why this is disabled? something like `We are stopping users from enabling 2FA on Near Wallet as part of wallet migration requirement` (I hope we get some help on rephrase this to better wordings)

Reply:
We just got confirmation that we should disable across MNW and Near-wallet